src/assets/cipher-card.css

```
@@ -86,6 +86,7 @@
 }
 
 :deep(.cipher-input),
+:deep(.cipher-select),
```

Stylelint (17.11.0)

[error] Unknown pseudo-class selector ":deep" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)